phoenix-core/src/main/java/org/apache/phoenix/iterate/TableResultIterator.java

```diff
@@ -115,6 +111,7 @@ public class TableResultIterator implements ResultIterator {
         this.scanGrouper = null;
         this.caches = null;
         this.retry = 0;
+        this.isMapReduceContext = true;
```

Comment:
I think this should be set to false now.

Reply:
Changed it Thanks!
I'll run tests once more locally, pipeline is failing because of test failing above.